compiler/resolution/wrappers.cpp

```diff
@@ -1467,14 +1493,37 @@ static void addArgCoercion(FnSymbol*  fn,
     checkAgain = true;
     castCall   = new CallExpr("readFE", gMethodToken, prevActual);
 
+    if (argumentCanModifyActual(formal))
+      issueErrorValueTemporaryToRef(call, formal);
+
   } else if (isSingleType(ats->getValType()) == true) {
     checkAgain = true;
 
     castCall   = new CallExpr("readFF", gMethodToken, prevActual);
 
+    if (argumentCanModifyActual(formal))
+      issueErrorValueTemporaryToRef(call, formal);
+
+  } else if (isManagedPtrType(ats->getValType()) == true &&
```

Comment:
I know that this code is already full of these, but comparing `boolean == true` ='(